RussianFilter/sections/specific.txt

```diff
@@ -515,6 +515,7 @@ oxax.tv##.len_kan
 ||oxax.tv/len.php
 reshimvse.com###addpc_after_solution
 /reshimvse\.com\/img\/\d+[a-z]?\./$image,domain=reshimvse.com
+mamalisa.com##div[style^="text-align:right;padding:0px;"]
```

Comment:
Why Russian filter? You block one of leftovers, better something like `mamalisa.com##.row div[class] > div[style]:contains(Advertisement)`